Description: Add missing "include"
 Add #include <string.h> to fix the warnings: "implicit
 declaration of function 'memcpy'" and "implicit declaration of function
 'memset'"
Author: Sophie Brun <sophie@freexian.com>
Last-Update: 2016-03-11
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- common/lib/r_assoc.c
+++ common/lib/r_assoc.c
@@ -56,6 +56,7 @@
 
 static char *RCSSTRING="$Id: r_assoc.c,v 1.4 2001/12/24 06:06:26 ekr Exp $";
 
+#include <string.h>
 #include <r_common.h>
 #include "r_assoc.h"
 
--- common/lib/r_bitfield.c
+++ common/lib/r_bitfield.c
@@ -10,6 +10,7 @@
 
 static char *RCSSTRING="$Id: r_bitfield.c,v 1.3 2001/12/24 06:06:26 ekr Exp $";
 
+#include <string.h>
 #include <r_common.h>
 #include "r_bitfield.h"
 
--- common/lib/r_data.c
+++ common/lib/r_data.c
@@ -46,6 +46,7 @@
 
 static char *RCSSTRING="$Id: r_data.c,v 1.3 2001/07/20 23:33:15 ekr Exp $";
 
+#include <string.h>
 #include <r_common.h>
 #include <r_data.h>
 
